mslib/mscolab/file_manager.py

```diff
@@ -103,7 +104,11 @@ def create_operation(self, path, description, user, last_used=None, content=None
             if content is not None:
                 operation_file.write(content)
             else:
-                operation_file.write(mscolab_settings.STUB_CODE)
+                content = mscolab_settings.STUB_CODE
```

Comment:
we want to remove the constant STUB_CODE and use only the new_flighttrack_template which the user provides in her msui_settings.json